k8s/sonarqube/chart/sonarqube/templates/prometheus-to-sd.yaml

```diff
@@ -0,0 +1,40 @@
+{{ if .Values.metrics.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-sonarqube-metrics-exporter
+  labels: &SonarqubeDeploymentLabels
+    app.kubernetes.io/name: {{ .Release.Name }}
+    app.kubernetes.io/component: sonarqube-metrics-exporter
+spec:
+  selector:
+    matchLabels: *SonarqubeDeploymentLabels
+  replicas: 1
+  template:
+    metadata:
+      labels: *SonarqubeDeploymentLabels
+    spec:
+      containers:
+      - name: prometheus-to-sd
+        image: {{ .Values.metrics.image }}
+        ports:
+        - name: profiler
+          containerPort: 6060
+        command:
+        - /monitor
+        - --stackdriver-prefix=custom.googleapis.com
+        - --source={{ .Release.Name }}:http://{{.Release.Name}}-{{ .Values.sonarqube.service.name}}:9000/api/prometheus/metrics
```

Comment:
You are missing Postgresql specific exporter source

Reply:
fixed